src/VirtoCommerce.ElasticAppSearch.Core/Models/Api/Search/SearchResult.cs

```diff
@@ -0,0 +1,12 @@
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace VirtoCommerce.ElasticAppSearch.Core.Models.Api.Search;
+
+public record SearchResult
+{
+    public SearchResultMetadata Meta { get; set; }
+
+    [JsonProperty(NamingStrategyType = typeof(DefaultNamingStrategy))]
+    public SearchResultDocument[] Results { get; set; }

```

Comment:
consider initializing collection win empty lists

Reply:
It will be deserialized from json, so it's redundant